frontend/ancient-stone-circles-india/style.css

```
@@ -0,0 +1,129 @@
+body {
+    background-color: var(--bg-primary, #1a1a1a);
+    color: var(--text-primary, #e0e0e0);
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)